src/Tizen.Wearable.CircularUI.Chart.Forms.Renderer/GroupBarChartViewRenderer.cs

```diff
@@ -0,0 +1,406 @@
+        SKColor[] _barBackgroundColor;
+
+        public GroupBarChartViewRenderer()
+        {
+        }
+
+        protected override void OnElementChanged(ElementChangedEventArgs<BarChartView> e)
+        {
+            base.OnElementChanged(e);
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            base.Dispose(disposing);
+        }
+
+        protected override void DrawContent(SKCanvas canvas)
+        {
+            if (Element == null || Element.Data == null)
+            {
+                Log.Debug(FormsCircularUIChart.Tag, "GroupBar Chart Element == null || Element.Data == null");
+                return;
+            }
+
+            if (Element.AxisOption != null)
+            {
+                base.CalculateAxisSize(canvas);
+            }
+            else
+            {
+                _majorAxisSize = new SKSize(0, 0);
+                _minorAxisSize = new SKSize(0, 0);
+                _referenceItemSize = new SKSize(0, 0);
+            }
+
+            var barSize = this.CalculateBarSize();
+            GenerateDataTable();
+            var points = CalculatePoints(barSize);
+            if (Element.AxisOption.IsVisibleOfMajorAxisLine)
+            {
+                base.DrawMajorAxisLine(canvas);
+            }
+
+            if (_categoryLabelCount > 0)
+            {
+                DrawCategoryLabels(canvas, points);
+            }
+
+            if (Element.AxisOption.IsVisibleOfMinorAxisLine)
+            {
+                base.DrawMinorAxisLine(canvas);
+            }
+
+            if (Element.AxisOption.IsVisibleOfReferenceLabel)
+            {
+                base.DrawReferenceLabels(canvas, barSize);
+            }
+
+            DrawGroupBarBackground(canvas, points, barSize);
+
+            if (Element.AxisOption.IsVisibleOfReferenceLine)
+            {
+                base.DrawReferenceLines(canvas, barSize);
+            }
+
+            DrawGroupBars(canvas, points, barSize);
+        }
+
+        int GetMaxEntriesCount()
+        {
+            int entriesMaxCount = 0;
+            for (int i = 0; i < _dataSetArrayCount; i++)
+            {
+                var dataSet = Element.Data.DataItemGroups[i];
+                int entryCount = dataSet.DataItems.Count();
+                if (entriesMaxCount < entryCount)
+                    entriesMaxCount = entryCount;
+            }
+
+            return entriesMaxCount;
+        }
+
+        protected override void GenerateDataTable()
+        {
+            _groupBarChartDataTable = new double[_groupDataCount, _categoryCount];
+            for (int i = 0; i < _groupDataCount; i++)
+            {
+                if (Element.Data.DataItemGroups.Count <= i) break;
+                var items = Element.Data.DataItemGroups[i].DataItems;
+                for (int j = 0; j < items.Count; j++)
+                {
+                    _groupBarChartDataTable[i, j] = items[j]?.Value ?? Element.Minimum;
```

Comment:
if items.Count was smaller than _categoryCount, 
`_groupBarChartDataTable[X,N]` not assigned as Element.Minimum value when N was larger than items.Count

Reply:
Done